### DISCUSSION TITLE
Is it possible to render html generated by the LLM within the chat?

### DISCUSSION THREAD

**Discussion by jabbasj** [open-webui/open-webui#4101] [Q&A]
What I'm trying to do is :
- render collapsible sections
- generate buttons
- render a plotlyjs graph

I tried 'details' tag and 'pre' tag but no luck.

I saw in the code (ResponseMessage.svelte) that code blocks and especially mermaid sections are spared from marked.parse... is that the way? I need to create my own exception?

**Comment by Yanyutin753** [comment]
#3988

**Reply by jabbasj** [reply]
That's quite interesting, I'll take it 👍

**Reply by tongbaoloc** [reply]
Could you please provide an update on the feature currently in progress? tks
